dashboard/db/migrate/20160315195700_associate_answers_with_modules_not_tasks.rb

```diff
@@ -0,0 +1,9 @@
+class AssociateAnswersWithModulesNotTasks < ActiveRecord::Migration
+  def change
+    #At the time this migration was written there were no Evaluation Answers in production, so it's safe to blow
+    #everything up
+    Plc::EvaluationAnswer.destroy_all
```

Comment:
but is it necessary? Why not just change the columns without clearing the table?